### ISSUE TITLE
Render Messages and Art Cards On Homepage

### ISSUE THREAD

**Issue by zapplecat** [Manotomo-Alliance-Support-Squad/holo-valentine-2021#4]
To be done after #1 

Both message and art cards should be rendered together on the home page. Preferably rendered in some sort of psudo random way.

**Comment by zapplecat** [comment]
I'll just do this alongside of #1 since I'm implementing art cards to use base classes anyways.

**Comment by zapplecat** [comment]
I'm addressing this as a part of #18 but will need another PR to do the rendering via a single combined section render. I'm not going to worry about this too much for now since we should get #18 reviewed and done before this gets more attention.

There's always the fall back option to create a gallery page anyways.

**Comment by zapplecat** [comment]
I must've accidentally closed this.
